components/wpt-results.html

```diff
@@ -138,12 +138,20 @@
           testRuns: {
             type: Array,
           },
+          filteredDirectories: {
+            type: Array,
+            value: []
+          },
         };
```

Comment:
Not important, but I just noticed a semicolon here and below (after super.connectedCallback()).

Probably not worth bothering about now, but anyway this just reminded me that setting up linting would be neat.